src/components/common/MemosTable/AddReply/Reply.js

```diff
@@ -13,6 +13,7 @@ function ReplyInput(props) {
   const { note_id } = props;
   const [formValues, setFormValues] = useState({ comment_text: '' });
   // const [comments, setComments] = useState({ comment_text: '' });
+  const { axiosWithAuth } = useAxiosWithAuth0();
```

Comment:
Is this creating a custom hook for axiosWithAuth?

Reply:
Yep, a custom hook that exports the function that we've been using so far. It's just being wrapped in a hook so that we can pull in the useAuth0 hook for the token.